### ISSUE TITLE
jx sync uses an incorrect image

### ISSUE THREAD

**Issue by vfarcic** [jenkins-x/jx#7015]
### Summary 

The image `vaporio/ksync:0.4.4` is no longer available.

### Steps to reproduce the behavior

```bash
jx sync --daemon
```

The output:

```
Downloading https://github.com/vapor-ware/ksync/releases/download/0.4.4/ksync_darwin_amd64 to /Users/vfarcic/.jx/bin/ksync.tmp...
Downloaded /Users/vfarcic/.jx/bin/ksync.tmp
Initialising ksync
==== Preflight checks ====
Cluster Config                              ✓
Cluster Connection                          ✓
Cluster Version                             ✓
Cluster Permissions                         ✓

==== Cluster Environment ====
Adding ksync to the cluster                 ✓
Waiting for pods to be healthy              ✘
↳       Timed out waiting for the ksync daemonset to be healthy.

To debug, you can:
- Run 'kubectl --namespace=kube-system --context= get pods -lapp=ksync' to look at what's going on.
- Run 'ksync doctor' to do an in-depth check of your system and the cluster.
FATA[0080]                                              
ERROR: Error: Command failed  ksync init --upgrade --image vaporio/ksync:0.4.4
error: exit status 1
```

The last semantic tag is `0.3.6`, while the latest tag is `git-c5fd87d`

### Expected behavior

It works.

### Actual behavior

It doesn't work

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 2.0.1249
jenkins x platform 2.0.2112
Kubernetes cluster v1.14.10-gke.27
kubectl            v1.16.7
helm client        2.14.3
git                2.24.1 (Apple Git-126)
Operating System   Mac OS X 10.15.3 build 19D76
```

### Jenkins type

- [X] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

### Kubernetes cluster

GKE

### Operating system / Environment

macOS

**Comment by nodece** [comment]
It seems have some `Node` is not working in your cluster.

**Comment by vfarcic** [comment]
If I describe the Pod, it says that it cannot pull the image. That makes sense since the tag `0.4.4` does not exist in https://hub.docker.com/r/vaporio/ksync/tags.

**Comment by hferentschik** [comment]
I think this is related to this pull request https://github.com/jenkins-x/jx/pull/6919

**Comment by ankitm123** [comment]
@vfarcic this should be fixed by #6919, are you still seeing the issue? I will add some integration tests as discussed with @hferentschik some time soon.
